### PR TITLE
Add update-rc.d-0.8 documents

### PR DESCRIPTION
**Pull request by ParianGol**
This package has only one file and that one file has a license

### CODE REVIEW

File: analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.rdf.xml
Comment (OliverFendt) [inline analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.rdf.xml:1]: This file contains 4 times the text of the CC0-1.0 license, I assume that somethin went wrong when you generated the rdf file. In contrast to the other spdx files the rdf file contains the text of the data license (CC-1.0) but not 4 times. Did you use https://tools.spdx.org/app/convert/ for generating the rdf file?
Please check this

File: analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.yaml [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+    \    that may apply to the Work or any use thereof, including without\n    limitation\
+    \ any person's Copyright and Related Rights in the Work.\n    Further, Affirmer\
+    \ disclaims responsibility for obtaining any necessary\n    consents, permissions\
+    \ or other rights required for any use of the\n    Work.\n d. Affirmer understands\
+    \ and acknowledges that Creative Commons is not a\n    party to this document\
+    \ and has no duty or obligation with respect to\n    this CC0 or use of the Work.\n\
+    \ "
+  licenseId: LicenseRef-CC0-1.0
+  name: Creative Commons Zero v1.0 Universal
+- extractedText: " GNU GENERAL PUBLIC LICENSE\n\nVersion 2, June 1991\n\nCopyright\
```
Comment (OliverFendt) [inline analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.yaml:131]: this item has as licenseId LicenseRef-CC0-1.0 but the extracted text is the text of the GPL-2.0
I assume that something went wrong during the generation of this file. Did you use https://tools.spdx.org/app/convert/
I recommend either to use the spdx online tools or the spdx java tools for generation of the other spdx formats.

**Review by OliverFendt — COMMENTED** [review]
@ParianGol great contribution, thank you very much, I only have some comments concerning the different SPDX formats, the disclosure file is fine.
Can you please check my comments and work on them? Thank you

File: analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.yaml (hunk 1 above)
Comment (ParianGol) [reply analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.yaml:34]: @OliverFendt Thank you very much for your thorough review! I have learned a lot what are the important points to keep eyes on. I have regenerated updated the files did the following changes:
1.  I have reported the issue regarding LicenseRef-CC0-1.0 to my colleagues. For the time being, I have manually removed those text related to this license
2. I have **regenerated** the files using the tool that I have written that uses spdx python tool **https://github.com/spdx/tools-python**. Now it does not have any issue as far as I know. I will keep an eye on it for the future (Thanks again)

**Review by OliverFendt — APPROVED** [review]
@ParianGol great, thank you very much for the contribution and all your efforts. Very well done. A very warm welcome to the group of contributors. I am looking forward to receive much more contributions from you.

**Pull request merged**